product_management/customer_requests_workflow.md

```diff
@@ -1,33 +1,36 @@
-# Customer requests workflow
-![](customer_requests.png)
+# Manage requests and defects
 
-Every request must be tracked in PRISMA. If possible, customers should be trained to submit new requests directly in the Trello board, tipically in a column called `(r) requests to be discussed`. If not possible, the PM will manually add requests here as they come.
 
-The `(r) ` at the beginning of a column name tells PRISMA to treat that column as a request column. This adds a feature to let you open a GitHub issue directly from the card maintaining traceability.
+
+![](/assets/manage_requests_and_defects.png)
+
+Every request must be tracked in PRISMA. If possible, customers should be trained to submit new requests directly in the Trello board, typically in a column called `requests to be refined`. If not possible, the PM will manually add requests here as they come.
+
+The `(r)` at the beginning of a column name tells PRISMA to let you open a GitHub issue directly from the card, maintaining traceability.
 
 Typically we have 4 requests columns:
 
-- `(r) icebox` requests that are on hold, buildo does not work on these
-- `(r) requests to be discussed` requests that need to be refined and transformed into actionable requests
-- `(r) bug reports` bug reports from the customer are just a special kind of request to be discussed, but it's useful to keep them apart
-- `(r) actionable requests` requests ready to go to the next step (backlog grooming)
+* `icebox` requests that are on hold, buildo does not work on these
+* `requests to be refined` requests that need to be refined and transformed into actionable requests
+* `bug reports` bug reports that need to be verified \(can come from the customer, the PM or the DEV team\)
+* `(r) actionable requests` requests ready to be prepared for development, kept here in a prioritized list
```

Comment:
è sottointeso che è compito del PM mantenerli `prioritized`. lo esplicitiamo/ripetiamo?

il mio punto è: visto che abbiamo vari attori (dev team, tech lead, cliente, PM) per rrendere la cosa meno confusa forse è meglio ripetere i concetti una volta in più anche se dovrebbero essere già sottointesi a questo punto del book